resources/js/frontend/wc-payplug-apple_pay_cart-blocks.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { getSetting } from '@woocommerce/settings';
+const settings = getSetting( 'apple_pay_data', {} );
+
+const ApplePayCart = ( props ) =>{
+	return (
+		<>
+			<div id="apple-pay-button-wrapper">
+				<apple-pay-button
+					buttonstyle="black"
```

Comment:
is it a good practice to hard-code the black style?

Reply:
In this place yes, because it's a required style for ApplePay